firefox_puppeteer/api/places.py

```diff
@@ -89,13 +89,15 @@ def restore_default_bookmarks(self):
               let observer = {
                 observe: function (aSubject, aTopic, aData) {
                   Services.obs.removeObserver(observer, "bookmarks-restore-success");
+                  Services.obs.removeObserver(observer, "bookmarks-restore-failed");
```

Comment:
nit: exchange those two lines just to keep the alphabetical order of removed observers.